frontend/public/components/storage-class-form.tsx

```diff
@@ -708,7 +708,7 @@ class StorageClassFormWithTranslation extends React.Component<
     const dataParameters = _.pickBy(
       _.mapValues(this.state.newStorageClass.parameters, (value, key) => {
         let finalValue = value.value;
-        if (this.storageTypes[type].parameters[key].format) {
+        if (this.storageTypes[type].parameters[key]?.format) {
```

Comment:
Exact steps are little harder to reproduce, but, this line was causing `Cannot read property 'format' of undefined` error for certain cases.
Anyways, it is safer to use `?.` than not adding it at all.

Reply:
@rawagner PTAL